src/Components/Web/src/Virtualization/VirtualizeBase.cs

```diff
@@ -0,0 +1,137 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information. 
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Components.Rendering;
+using Microsoft.JSInterop;
+
+namespace Microsoft.AspNetCore.Components.Virtualization
+{
+    /// <summary>
+    /// Provides common functionality for virtualized lists.
+    /// </summary>
+    /// <typeparam name="TItem">The <c>context</c> type for the items being rendered.</typeparam>
+    public abstract class VirtualizeBase<TItem> : ComponentBase, IVirtualizeJsCallbacks, IAsyncDisposable
+    {
+        private VirtualizeJsInterop _jsInterop = default!;
```

Comment:
Could this be more honest about the nullability? e.g.,

```suggestion
        private VirtualizeJsInterop? _jsInterop;
```

I know this might impact other code. If this is too disruptive then I understand!